app/backend/src/couchers/db.py

```diff
@@ -177,9 +177,24 @@ def new_password_reset_token(session, user, hours=2):
     return password_reset_token, f"{hours} hours"
 
 
-def set_email_change_token(session, user, hours=2):
+def set_change_token_old_email(session, user, hours=2):
```

Comment:
`set_email_change_token_for_{old/new}_email`

Reply:
If we make it one process, then yes, this can be combined. Otherwise it makes more sense to keep separate.